### ISSUE TITLE
What data should be inputted to the speedster optimizer?

### ISSUE THREAD

**Issue by orestmalinovskyi-digica** [nebuly-ai/nebullvm#172]
In all the example notebooks for PyTorch I see you use random tensors with labels of just zero. Nowhere it's specifically said what data should be used and in the documentation only once there's a general example on how to use torch dataloaders. 

So I have a question: what is the expected input data to the optimizer? Can be anything as long as it goes through all layers of the model, or should it be a special calibration dataset with true data and true labels? Or is it in a manner that if for example you don't specify metric drop threshold, then the input can be any random data, but if you specify it, then you should provide with the true data and labels?

**Comment by orestmalinovskyi-digica** [comment]
To be even more specific, the question is, can we use random data for optimizers - as in the examples, or do we have to use actual data?

**Comment by diegofiori** [comment]
Yes, you can use random data for optimizers, but you should get better performance using real data.

**Comment by diegofiori** [comment]
Do you have any suggestion for improving the docs? Very much appreciated

**Comment by orestmalinovskyi-digica** [comment]
thanks for the swift answer! would has been great to see this in the docs in the first place.
